src/ducks/apps/components/ApplicationPage/Gallery.jsx

```diff
@@ -3,12 +3,34 @@ import React from 'react'
 import { translate } from 'cozy-ui/react/I18n'
 
 export const Gallery = ({t, slug, images}) => {
+  if (!images) return null
+  const imagesParts = images.reduce((acc, image, index) => {
```

Comment:
I think you can totally do this without splitting the images in to part, but with a smart flex layout, who would be much easier to decline for responsive version !